source/javascripts/controllers/_MainController.js.erb

```diff
@@ -674,11 +691,12 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 				launchDarklyService.initialize().then(() => {
 					viewModel.guidedOnboardingEnabled = launchDarklyService.variation('enable-guided-onboarding');
 					if (viewModel.guidedOnboardingEnabled) {
-						const metadataReq = requestService.getCurrentUserMetadata('show_getting_started_guide');
+						const guideMetadataReq = requestService.getCurrentUserMetadata('show_getting_started_guide');
+						const lastWFEDateMetadataReq = requestService.getCurrentUserMetadata('last_workflow_edited_date');
 						const tourReq = requestService.getProductTourShown();
-
-						Promise.all([metadataReq, tourReq]).then(([metadataResp, tourResp]) => {
-							viewModel.guidedOnboardingEnabledForUser = metadataResp;
+						Promise.all([guideMetadataReq, lastWFEDateMetadataReq, tourReq]).then(([guideMetadataResp,lastWFEDateMetadataResp, tourResp]) => {
+							viewModel.guidedOnboardingEnabledForUser = guideMetadataResp;
+							viewModel.lastWorkflowEditedDate = lastWFEADateMetadataResp;
```

Comment:
```suggestion
							viewModel.lastWorkflowEditedDate = lastWFEDateMetadataResp;
```

Reply:
same thought as https://github.com/bitrise-io/bitrise-workflow-editor/pull/716/files#r1010892542
Can we confirm the response is parsed into a `Date` object?

Reply:
looks like the above suggestion still holds true. I think you will always get `viewModel.lastWorkflowEditedDate` as `undefined` if not a run time error

Reply:
Sorry about the confusion, I didn't properly respond to your feedback here. The response is actually a string, and my last changes reflect this confirmation.